pkg/proxy/apis/config/validation/validation.go

```diff
@@ -85,9 +85,6 @@ func Validate(config *kubeproxyconfig.KubeProxyConfiguration) field.ErrorList {
 			if err != nil || !isDual {
 				allErrs = append(allErrs, field.Invalid(newPath.Child("ClusterCIDR"), config.ClusterCIDR, "must be a valid DualStack CIDR (e.g. 10.100.0.0/16,fde4:8dba:82e1::/48)"))
 			}
-		// if not DualStack only one CIDR allowed
-		case len(cidrs) > 1:
```

Comment:
Previously here contained another condition for `dualStackEnabled` but that has been removed since the commit a53e2eaeaba064309dceca2dc27f3ac09c6375b0
Then this meaningless check is still there.